backend/users/src/app/resolvers/people.resolver.ts

```diff
@@ -0,0 +1,102 @@
+import { Query, Resolver, Mutation, Args, Int } from '@nestjs/graphql';
+
+import { AuthenticatedUser, Public, Resource } from 'nest-keycloak-connect';
+import { CreatePersonInput } from '../dto/createPersonInput';
+import { NotFoundException } from '@nestjs/common';
+import { SearchPersonResponse } from '../dto/reponse/fetchSearchPerson';
+import { UpdateExternalUserResponse } from '../dto/reponse/updateExternalUserResponse';
+import { LoggerService } from '../logger/logger.service';
+import { PersonService } from '../services/people.service';
+import { Person } from '../entities/person.entity';
+
+@Resolver(() => Person)
+@Resource('user-service')
+export class PersonResolver {
+  constructor(
+    private readonly personService: PersonService,
+    private readonly loggerSerivce: LoggerService,
+  ) {}
+
+  @Query(() => [Person], { name: 'findAllPerson' })
+  async findAll() {
+    try {
+      return await this.personService.findAll();
+    } catch (error) {
+      throw new Error(`Failed to fetch person: ${error.message}`);
+    }
+  }
+
+  @Query(() => Person, { name: 'findPersonById' })
+  async findOne(@Args('id') id: number) {
+    try {
+      const person = await this.personService.findOne(id);
+      if (!person) {
+        throw new NotFoundException(`Person with ID ${id} not found`);
+      }
+      return person;
```

Comment:
make sure the api response is consistent with what we have done for site api's

Reply:
Mostly Jaspal will be updating this code, as he is working on this feature.